src/components/layout/nav/Navbar.jsx

```diff
@@ -159,7 +163,10 @@ const Navbar = () => {
         <div className="iconWrapper">
           <i
             className="fas fa-user-circle"
-            onClick={() => handleStatus("IS_SIGNUP_OPENED")}
+            onClick={() => {
+              handleStatus("IS_SIGNUP_OPENED");
```

Comment:
you should never be using magic strings like this, actually i should have corrected this earlier

generally we would move this into a constant file and export it, reusing it everywhere

for example see how for redux types they just declare a string cosntant

https://redux.js.org/basics/actions

you would do something similar here, otherwise it is typo-prone to be typing strings manually like this

Reply:
( i know this isn't a redux type but i'm using it as an example)

Reply:
Definitely! I moved them to a constant now.